### ISSUE TITLE
Bug: TypeError: Cannot read property 'removeEventListener' of undefined

### ISSUE THREAD

**Issue by pkarw** [SSENSE/vue-carousel#139]
Hi Guys,

We're using vue-carousel in https://github.com/DivanteLtd/vue-storefront

We've recently (0.6.7?) got strange error when page is destroyed:

![zrzut ekranu 2018-02-28 08 59 46](https://user-images.githubusercontent.com/211899/36776381-d95ea93a-1c65-11e8-97ea-a5e31a157dc6.png)

Our case is pretty simple, and the carousel works just fine - but the error above:

```js
<template>
  <section class="main-slider w-100 bg-darkgray c-white">
    <no-ssr>
      <carousel :per-page="1" pagination-active-color="transparent" pagination-color="#F2F2F2">
        <slide v-for="(slide, index) in slides" :key="index">
          <div class="container w-100" :style="{ backgroundImage: 'url(' + slide.image + ')' }">
            <div class="row middle-xs center-xs">
              <div class="col-md-12 px10p">
                <p class="subtitle mb0 serif uppercase h3 align-center">{{ slide.subtitle }}</p>
                <h1 class="title mt0 mb30 align-center">{{ slide.title }}</h1>
                <div class="align-center">
                  <button-outline :link="slide.link" color="light">
                    {{ slide.button_text }}
                  </button-outline>
                </div>
              </div>
            </div>
          </div>
        </slide>
      </carousel>
    </no-ssr>
  </section>
</template>
```

It seems like a problem with https://github.com/SSENSE/vue-carousel/blob/2a5018b89a859b1af5771262a5cff1c8190b4bc5/src/Carousel.vue#L666 

BUT no idea why :)

**Comment by Tr4X** [comment]
Hi, same error here, the carousel is working just fine but I a console error is generated as soon as I leave the page where the carousel was on.

<img width="825" alt="capture d ecran 2018-02-28 a 10 28 06" src="https://user-images.githubusercontent.com/1759467/36780058-2c1ab4d2-1c72-11e8-849f-bf78be0f78cf.png">

**Comment by JensvdHeydt** [comment]
Same here using nuxtjs
![image](https://user-images.githubusercontent.com/19750185/36782112-2c837840-1c78-11e8-8dee-85384eec5354.png)

**Comment by quinnlangille** [comment]
Hey guys, is everyone using nuxt or some form of SSR with this issue? It could potentially be from a missing class that got hot fixed in v0.6.8 so try rolling up to that and see if ti helps. Also, is it breaking anything or just throwing errors? Regardless, I'll look into it today and add a fix

**Comment by quinnlangille** [comment]
Also, if any of you guys manage to fix it on your end, feel free to submit a PR with the solve! At first glance, it just looks like a missing conditional check on the remove event listener

**Comment by VinceG** [comment]
@quinnlangille Submitted PR, very simple fix hoping it can get merged today since we are also affected by this.

Thank You.

**Comment by quinnlangille** [comment]
@VinceG just merged! Will release it with another change before the end of the day

**Comment by quinnlangille** [comment]
Just released 0.6.9 with the fix! Thanks guys 🎉

**Comment by pkarw** [comment]
Wow, guys didn't expect so quick response! Thanks a lot, great job! :)
